terms/bayesianmodels_kj.bib

```diff
@@ -0,0 +1,25 @@
+@book{Manning2009,
```

Comment:
In general, few references ... but I understand your entry is mostly textbook material.